src/Microsoft.ML.StandardTrainers/FactorizationMachine/FactorizationMachineTrainer.cs

```diff
@@ -81,7 +81,7 @@ public sealed class Options : TrainerInputBaseWithWeight
             /// Whether to normalize the input vectors so that the concatenation of all fields' feature vectors is unit-length.
             /// </summary>
             [Argument(ArgumentType.AtMostOnce, HelpText = "Whether to normalize the input vectors so that the concatenation of all fields' feature vectors is unit-length", ShortName = "norm", SortOrder = 6)]
-            public bool Normalize = true;
+            public new bool NormalizeFeatures = true;
```

Comment:
Why do we need to hide it? Could we set the underlying field to always be `false` and expose `Normalize` as is?

Reply:
then there will be 2 fields in manifest.json and confusion over which Normalize* to use. 

---
In reply to: [266176252](https://github.com/dotnet/machinelearning/pull/2964#discussion_r266176252) [](ancestors = 266176252)

Reply:
No one cares about entry points this and next weeks, right?

---
In reply to: [266176427](https://github.com/dotnet/machinelearning/pull/2964#discussion_r266176427) [](ancestors = 266176427,266176252)

Reply:
yes its difficult to filter out hidden fields, but its currently not needed in this PR. It would be good to fix this as NimbusML depends on manifest.json. This duplication will propagate confusion in set of params for FFM class in NimbusML as well as docs.  

---
In reply to: [266176837](https://github.com/dotnet/machinelearning/pull/2964#discussion_r266176837) [](ancestors = 266176837,266176427,266176252)